rest-api/dao/dv_order_dao.py

```diff
@@ -120,11 +127,11 @@ def from_client_json(self, resource_json, id_=None, expected_version=None,
         order.id = self.get_id(order)[0]
         order.created = self._get_created_date(participant_id, id_)
         order.version = expected_version
-        order.barcode = resource['barcode']
+        order.barcode = fhir_resource.barcode  # NOTE: not in the FHIR spec
```

Comment:
This line will behave the same as before this change, but `barcode` is not present in the example or in the FHIR spec for `SupplyRequest`